.github/workflows/release-azure-webapp-v1.yaml

```diff
@@ -0,0 +1,268 @@
+on:
+  workflow_call:
+    inputs:
+      workspace_name:
+        description: The name of the workspace to create the artifact for.
+        type: string
+        required: true
+      resource_group_name:
+        description: Web App resource group name.
+        type: string
+        required: true
+      web_app_name:
+        description: Web App name.
+        type: string
+        required: true
+      environment:
+        description: Environment where the artifact will be deployed.
+        type: string
+        required: true
+      use_staging_slot:
+        description: Use a staging slot for the deployment.
+        type: boolean
+        required: false
+        default: true
+      disable_auto_staging_deploy:
+        description: Disable automatic deployments to the staging slot.
```

Comment:
question: do you mean "automatic swaps"? I guess the code is deployed to the staging slot anyway when using `use_staging_slot`.

Reply:
`auto_approve_staging_deploy` is a better name

Reply:
@gunzip I mean automatic deploy, not automatic swap. This workflow by default have a job that approves automatically the deploy to staging slot. 

In the previous version of the workflow (`web_app_deploy`) it was an _opt-in_ feature, not I changed it to be opt-out.